cmd/snap-mgmt/snap-mgmt-selinux.sh.in

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
```

Comment:
I think this file needs to include the license.

Reply:
Looked around the tree, and contrary to Go and C code, none of the scripts under `/cmd` seem to have a license.